extension.js

```diff
@@ -201,11 +201,15 @@ export default class HardDiskLEDExtension extends Extension {
         this.button.set_child(this.layoutManager);
 
         Main.panel._rightBox.insert_child_at_index(this.button, 0);
-        this.timeout = Mainloop.timeout_add_seconds(this.refreshTime, this.parseStat.bind(this));
+        this.timeout = GLib.timeout_add(
+            GLib.PRIORITY_DEFAULT, 
+            this.refreshTime, 
```

Comment:
multiply by 200 here?

Reply:
The multiplication by 200 was implemented to adjust the update interval, aiming to improve user experience. Originally, the update timing was set in such a way that the read or write values were updated too quickly, leading to unstable and sometimes uncomfortable viewing for the user.

These values often appeared and disappeared almost instantaneously, making it difficult to read and interpret the data in real-time. By adjusting the update interval to a slightly longer period (by multiplying by 200), we achieved an update rate that allows for a more stable and understandable display of values.